src/app/search/directives/search-story.component.css

```diff
@@ -0,0 +1,121 @@
+.story-image {
+  position: relative;
+  float: left;
+  width: 120px;
+  height: 120px;
+  margin-bottom: 5px;
+}
+image-loader {
+  width: 100%;
+  height: 100%;
+  display: block;
+}
+
+.status {
+  position: absolute;
+  top: 0;
+  right: 0;
+  padding: 3px 10px;
+  z-index: 10;
+  color: #fff;
+  background-color: #444;
+  font-weight: 400;
+  font-size: 11px;
+  text-transform: uppercase;
+  line-height: 18px;
+}
+.status.unpublished {
+  background-color: #ee4d3e;
+}
+
+.story-detail {
+  position: relative;
+  float: right;
+  padding: 10px;
+  width: 200px;
+  height: 120px;
+  line-height: 20px;
+}
+.story-title {
+  font-size: 16px;
+  max-height: 40px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+.series-title {
+  max-height: 40px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.meta {
+  position: absolute;
+  bottom: 0;
+  width: 180px;
+  line-height: 20px;
+}
+.duration, .modified, .audio-total {
+  width: calc(100%/3);
+}
+.modified {
+  text-align: right;
+}
+.audio-total {
+}
+.audio-total > i {
+  /* TODO: this isn't the correct icon */
+  display: inline-block;
+  transform: rotate(90deg);
+  margin-right: -5px;
+}
+
+.story-tags {
+  clear: left;
+  display: flex;
+  justify-content: flex-start;
+  flex-flow: row wrap;
+  width: 320px;
+  padding: 0 10px;
+  max-height: 20px;
+  line-height: 20px;
+  overflow: hidden;
+}
+.story-tags > span {
+  background-color: #f0f0f0;
+  padding: 0 4px;
+}
+
+.story-tags > span + span {
+  margin-left: 4px;
+}
+
+.story-description {
+  clear: left;
+  width: 320px;
+  padding: 0 10px;
+  margin-top: 5px;
+  margin-bottom: 10px;
+}
+
+.three-lines {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  max-height: 60px;
+  line-height: 20px;
+  -webkit-line-clamp: 3;
+  display: block;
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
```

Comment:
Major and modern (current and last prior major release), so no old IE support pain.
h/t @farski, I couldn't remember where we wrote this down: https://help.prx.org/hc/en-us/articles/203763430-Supported-browsers

Should probably add this and other FE standards to: 
https://github.com/PRX/meta.prx.org/wiki/Best-Practices for browser testing requirement
and
https://github.com/PRX/meta.prx.org/wiki/Project-Standards needs more FE focused standards like browser support